onnx/defs/traditionalml/defs.cc

```diff
@@ -74,7 +74,18 @@ ONNX_OPERATOR_SCHEMA(CastMap)
         "max_map",
         "if map_form packing is SPARSE, what is the total length of each output in N (max index value)",
         AttributeProto::INT,
-        static_cast<int64_t>(1));
+        static_cast<int64_t>(1))
+    .TypeAndShapeInferenceFunction([](InferenceContext& ctx) {
+      auto& cast_to = ctx.getAttribute("cast_to")->s();
```

Comment:
getAttribute() can return null if the attribute is not specified. check for null (and do nothing if it is null). (In the longer run, we should probably add methods to InferenceContext so that we can return an error-code to indicate the exact typing-errors for better error messages.)

Reply:
fixed.